packages/@eventual/cli/src/commands/local.ts

```diff
@@ -177,11 +187,80 @@ export const local = (yargs: Argv) =>
       const app = express();
       const server = http.createServer(app);
 
+      /**
+       * Catches the url pattern of /__bucket/presigned/ and handles it like a s3 presigned url.
+       */
+      const presignedMiddleware: RequestHandler = async (req, res, next) => {
+        if (req.url.startsWith("/__bucket/presigned")) {
+          try {
+            const [, , , token] = req.url.split("/");
+            if (!token) {
+              res.status(400).send("Missing token");
+              return;
+            }
+            if (req.method === "PUT") {
+              const body = req.body
+                ? JSON.stringify(req.body)
+                : await getBody(req);
```

Comment:
This looks weird. Why are you stringifying the body?

Also, when I hacked around this, I had to be careful about handling OPTIONS pre-flight requests when calling put on the front end.

Reply:
Ohh, yeah, OPTIONS is a good point. I think I'll just need to let unhandled methods pass through to the normal handler.

I was stringifying here because this logic was originally AFTER the JSON body parser, but now it is before so I can remove this logic.